src/benchmark/ips.cr

```diff
@@ -153,6 +156,21 @@ module Benchmark
         @mean = samples.sum.to_f / size.to_f
         @variance = (samples.inject(0) { |acc, i| acc + ((i - mean) ** 2) }).to_f / size.to_f
         @stddev = Math.sqrt(variance)
+        @relative_stddev = 100.0 * (stddev / mean)
+      end
+
+      def human_mean
+        pair = case Math.log10(mean)
+               when 0..3
+                 {mean, ' '}
+               when 3..6
+                 {mean/1_000, 'k'}
+               when 6..9
+                 {mean/1_000_000, 'M'}
+               else
+                 {mean/1_000_000_000, 'G'}
```

Comment:
Usually I see the MM only when talking of money. I was basing the letters and their capitalization from https://en.wikipedia.org/wiki/Metric_prefix#List_of_SI_prefixes